api/capi/src/nnstreamer-capi-tizen.c

```diff
@@ -474,11 +543,12 @@ ml_tizen_mm_res_release (gpointer handle, gboolean destroy)
  * @brief Function to initialize mm resource manager.
  */
 static int
-ml_tizen_mm_res_initialize (ml_pipeline_h pipe)
+ml_tizen_mm_res_initialize (ml_pipeline_h pipe, gboolean has_video_src, gboolean has_audio_src)
 {
   ml_pipeline *p;
   pipeline_resource_s *res;
   tizen_mm_handle_s *mm_handle;
+  gboolean failed = FALSE;
```

Comment:
I think this flag is redundant because `rm_error` always handles failed cases.
In `rm_error`, checking whether `mm_handle` is NULL would be enough.

Reply:
You are right, I just changed to check the error case with ```status != ML_ERROR_NONE```